### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by TrySound**
es6-promise now ignores tests. So, package is much smaller.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -9,7 +9,7 @@ env:
 
 rules:
   indent: [2, 2] # 2 spaces indentation
-  max-len: [2, 80, 4]
+  max-len: [2, 100, 4]
```
Comment (MoOx) [inline .eslintrc]: Don't change a rule without asking. This is not the scope of the PR. Tahnks.

Comment (TrySound) [inline .eslintrc]: That's why you need to confirm PR, not just merge. It's asking.
eslint doesn't like this in test/index.js. That string has 100 chars.
```
(t,
arg,
arg)
```

**Comment by TrySound** [comment]
Does it better?

File: .eslintrc (hunk 1 above)
Comment (MoOx) [inline .eslintrc]: screen shot 2015-08-11 at 10 03 56
Not sure why you needed to change this...

Comment (TrySound) [inline .eslintrc]: There was a few extra spaces.

**Pull request merged**